crates/runtime/src/memory.rs

```diff
@@ -115,6 +171,7 @@ pub struct MmapMemory {
 
 impl MmapMemory {
     /// Create a new linear memory instance with specified minimum and maximum number of wasm pages.
+    /// TODO remove minimum/maximum; already contained within `plan`.
```

Comment:
IIRC the reason for passing them explicitly was that it's somewhat tricky to calculate the maximum/minimum converting from u32-or-u64 to `usize` so by passing them here it's "pre-validated" from the results of the previous calculations.